server/plugins/api/GenericNotifier.ts

```diff
@@ -0,0 +1,5 @@
+import { APIReturn } from "./APIReturn";
+
+export interface GenericNotifier<T> {
```

Comment:
nit. Like yeah its `Generic` don't need to call it a `GenericNotifier`, makes it sound more special than it is.
(you don't have to make this change)

Reply:
Gotta be enterprise webscale machine learning AI Y3K ready by being G E N E R I C